compiler/noirc_frontend/src/hir/comptime/interpreter.rs

```diff
@@ -1006,7 +992,7 @@ impl<'a> Interpreter<'a> {
         };
 
         if let Some(method) = method {
-            self.call_function(method, arguments, location)
+            self.call_function(method, arguments, TypeBindings::new(), location)
```

Comment:
Shouldn't we fetch instantiation bindings here? If we want this PR to just be functions and not method calls can leave a note and/or issue?

Reply:
It's a bit odd - method calls don't have instantiation bindings since they're replaced with normal function calls during type checking.

So this entire `evaluate_method_call` method is actually also unreachable. I thought about removing it for this PR but thought it would be too much / confusing / unrelated.

Reply:
Ok I see. Let's remove in a follow-up then